ground/openpilotgcs/src/plugins/setupwizard/vehicletemplateexportdialog.cpp

```diff
@@ -50,7 +50,7 @@
 
 const char *VehicleTemplateExportDialog::EXPORT_FIXEDWING_NAME = "fixedwing";
 const char *VehicleTemplateExportDialog::EXPORT_MULTI_NAME     = "multirotor";
-const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME      = "helicopter";
+const char *VehicleTemplateExportDialog::EXPORT_HELI_NAME = "helicopter";
```

Comment:
This changes comes from running "make uncrustify_ground", which we should run before pushing changes to next. Unfortunately, uncrustify indents the code like this. But I agree that it's not pretty, and I would have preferred if it was aligned differently.